test_data/rules/prefer_final_parameters.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
```

Comment:
Consider adding at least one method and one constructor inside a class. It wouldn't hurt to also have a getter, setter, and operator.

Consider having at least one example each of an optional positional, required named, and optional named parameter.

Reply:
Great ideas, thanks. Let me know what you think of the latest additions.